hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -188,6 +188,8 @@ class PrisonerSynchroniserService(
       prisonerDifferenceService.reportDiffTelemetry(existingPrisoner, prisoner, label)
 
       indices.map { prisonerRepository.save(prisoner, it) }
+
+      prisonerDifferenceService.generateDiffEvent(existingPrisoner, ob.offenderNo, prisoner, true)
```

Comment:
Is this the fix? Wasn't there a failing test for it?

Reply:
Yes, ill comment where that is